### PR TITLE
Add commits count to users page

### PR DESCRIPTION
**Pull request by Bounga**
This pull request adds number of commits integrated into the core for each contributor on its personal page

/cc @cllns

### CODE REVIEW

**Comment by cllns** [comment]
Thanks 🌸 . We discussed this in issue #40.

This PR should only include the changes relevant to this feature. Right now, it includes the commit you did for #41, too. Can you only include the relevant commit?

**Comment by Bounga** [comment]
Sure, sorry seems like I was a bit tired yesterday 😄

File: apps/web/controllers/contributors/show.rb [hunk 1]
```diff
@@ -3,11 +3,14 @@ class Show
...
 
-    expose :contributor, :commits
+    expose :contributor, :commits, :commits_count
 
     def call(params)
       @contributor = ContributorRepository.new.find_by_github(params[:id])
-      @commits = all_for_page(CommitRepository.new.all_for_contributor(@contributor.id))
+      commits = CommitRepository.new.all_for_contributor(@contributor.id)
+
+      @commits_count = commits.to_a.size
```
Comment (davydovanton) [inline apps/web/controllers/contributors/show.rb]: you can use faster way to get count :)
```
=> #<CommitRepository relations=[:commits]>
>> repo.root.count
[contributors] [INFO] [2018-02-07 12:07:03 +0300] (0.032178s) SELECT count(*) AS "count" FROM "commits" LIMIT 1
=> 6094
```

Comment (davydovanton) [reply apps/web/controllers/contributors/show.rb]: just call `commits.count`, because `commits` is relation as `repo.root`

Comment (Bounga) [reply apps/web/controllers/contributors/show.rb]: Nice, I didn't know if I could use `count` here. I should have tried. Thank you.

**Pull request merged**